packages/interface/src/components/action-item/README.md

```diff
@@ -0,0 +1,51 @@
+ActionItem
+=============================
+
+`ActionItem` is a component that implements a slot & fill pair used in situations where we have a container that will contain several possible actions e.g: a button group that will contain several buttons or a menu that will contain several menu items.
+
+## ActionItem.Slot
+
+The props not referred bellow are passed to the container component.
+
+## Props
+
+### name
+
+The name of the slot and fill pair passed to the `Slot` component.
+
+- Type: `String`
+- Required: Yes
+
+### bubblesVirtually
+
+Property used to change the event bubbling behavior, passed to the `Slot` component.
+
+- Type: `boolean`
+- Required: no
+
+### as
+
+An array of two components the first is the component used on the container of the fills and the second is the component used to render each fill.
```

Comment:
>It feels very limiting that the slot decides how to render fills

The slot is the component that is aware of the ideal fills to render, e.g: if the slot wraps things with ButtonGroup the ideal fill is Button, if the slot wraps things with MenuGroup the ideal fill is MenuItem, if the slot wraps things with ToolbarGroup the ideal fill is ToolbarButton.

But I understand your point it removes freedom from the fill. I updated the code to keep flexibility on the fill. Right now we allow the slot to set a default component to use in the fills but we also allow an as prop on the fill, in case a different component is desired.